lib/constants/cycles.ts

```diff
@@ -1,58 +1,94 @@
 /**
- * Liturgical Day cycle that can be used as metadata for liturgical days.
+ * Represent the two main cycle, that compose the liturgical year in the Roman rite.
  */
 export const ProperCycles = {
+  /**
+   * The "Proper of Time" (or "Temporal") consists of the movable feasts,
```

Comment:
`Temporal` → [`temporale`](https://en.wikipedia.org/wiki/Temporale), `Proper of Time` → `proper of time`

Note that _temporale_ is a common noun (i.e. it is not a proper noun, therefore the first letter is not capital usually) _unless it is used as a title of a book_. _Temporal_ (without final _e_) is an adjective in English. :wink:

_Proper of time_ is a noun phrase and consists of two common nouns.

Reply:
In this context, I'm referring to it as a title, to name one of these 2 liturgical cycle. Sanctorale or Temporale are also used to name specific lectionary books (although the Temporale is often split into 2 books: Sundays and Weekdays books).

I also want to keep capitalizing most words in titles (which is also a common practice in English), to emphasize important keywords.

However, I agree that if we use _Sanctorale_ (with the final "e"), to also write _Temporale_ with the final "e".

Reply:
Okay. Based on [this comment of yours](https://github.com/romcal/romcal/pull/343#issuecomment-1001069362) and [this comment of mine](https://github.com/romcal/romcal/pull/343#issuecomment-1001073463), I suggest to remove all mentions of `santorale` and `temporale` and use `Proper of {Saints, Time}` instead only.

Reply:
_Temporale_ and _Sanctorale_ can still be used in the liturgical vocabulary. Even if we use _Proper of Time_ or _Proper of Saint_ instead in romcal, I would like to mention these 2 words at least somewhere in the documentation. And here is a good place.

Reply:
Okay then, but use `temporale`/`sanctorale` instead of `temporal`/`sanctoral` as noun please.

```suggestion
   * The "Proper of Time" (or "Temporale") consists of the movable feasts,
```